### ISSUE TITLE
Throw the bearer utility into the generator

### ISSUE THREAD

**Issue by doroved** [rayriffy/elysia-rate-limit#32]
Is it possible to somehow throw the bearer utility into the generator?
```ts
import bearer from '@elysiajs/bearer'

const app = new Elysia()
  .use(cors())
  .use(bearer())
  .use(
    rateLimit({
      generator: (req) => {
        console.log(req.headers, bearer)
        return bearer
      },
    })
  )
  .use(v1)
  .listen(3000)
```

**Comment by rayriffy** [comment]
That would be great, so everyone can also uses `elysia-ip` plugin as key generator. Let me take a look.

**Comment by rayriffy** [comment]
From fast investigation, the answer is yes it's possible. But no (at this moment), I've to come up with TypeScript move to extends these derived values to third argument but I think I'm currently too dumb to achieve this.

I've made #33, and published [3.2.0-beta.2](https://www.npmjs.com/package/elysia-rate-limit/v/3.2.0-beta.2) for you to try (without proper types though, be warned).

**Comment by rayriffy** [comment]
Release at `3.2.0` (Ref: #34). Consult with README for details.
